client-java/controller/src/test/java/org/evomaster/client/java/controller/internal/db/ParserUtilsTest.java

```diff
@@ -42,4 +47,17 @@ public void testEmptyInsertValue(){
 
         assertTrue(parsed);
     }
+
+    @Test
+    public void testSelectOne() throws JSQLParserException {
+        String[] sqls = {"SELECT 1", "SELECT 1;", "Select    1;", "Select 1 ; "};
+        for (String sql: sqls){
```

Comment:
this could be refactored with a parameterized test. see use of `@ParameterizedTest`

Reply:
thanks. done